### DISCUSSION TITLE
Avoid xDrip to broadcast Calibrations values as BG Reading to AndroidAPS

### DISCUSSION THREAD

**Discussion by parapenT1sta** [NightscoutFoundation/xDrip#1934] [General]
Due the new security measures in AndroidAPS, if there are two glucose reading near to each other, the SMB and Close Loop stop working.
I made the report in AndroidAPS github page: https://github.com/nightscout/AndroidAPS/issues/1174
And an workaround was made in AndroidAPS for Dexcom App: https://github.com/nightscout/AndroidAPS/commit/0603e741be6c6cd23d2d4b6c84c70de528b63a43

Since I am also facing the same issue with xDrip, is it possible to avoid broadcast calibrations values to AndroidAPS?

**Comment by Navid200** [comment]
There is already an open issue:
https://github.com/NightscoutFoundation/xDrip/issues/1890
There is already a Nightly released to address it.
Tests are in progress to provide feedback to the developers if the fix has been successful or not.

**Reply by parapenT1sta** [reply]
@Navid200 Thank you for your feedback!
